Podfile

```diff
@@ -191,23 +191,7 @@ target 'WordPress' do
     end
 
     
-    post_install do
-      
-        ## Append SDKVersions.xcconfig contents to WordPressAuthenticator config files
-        Dir.glob("Pods/Target Support Files/WordPressAuthenticator/*.xcconfig") do |xc_config_filename|
```

Comment:
From the PR description, it looks like we would still need it because of WPAuthenticator, but looking at the code it seems that the new version without conditionals has been already imported, right?

Reply:
Right, if any pods are still using `#if XCODE11`, those will now fail because the condition will flip to false because we'll have removed the definition of Xcode 11.